src/mobot_client/models/__init__.py

```diff
@@ -117,21 +113,28 @@ class MessageDirection(models.IntegerChoices):
 
 
 class Message(models.Model):
-    customer = models.ForeignKey(Customer, on_delete=models.CASCADE)
+    customer = models.ForeignKey(Customer, on_delete=models.CASCADE, related_name="messages")
     store = models.ForeignKey(Store, on_delete=models.CASCADE)
     text = models.TextField()
     date = models.DateTimeField(auto_now_add=True)
     direction = models.PositiveIntegerField(choices=MessageDirection.choices)
 
 
+class OrderStatus(models.IntegerChoices):
+    STARTED = 0, 'started'
+    CONFIRMED = 1, 'confirmed'
+    SHIPPED = 2, 'shipped'
+    CANCELLED = 3, 'cancelled'
+
+
 class Order(models.Model):
-    customer = models.ForeignKey(Customer, on_delete=models.CASCADE)
-    drop_session = models.ForeignKey(DropSession, on_delete=models.CASCADE)
-    sku = models.ForeignKey(Sku, on_delete=models.CASCADE)
+    customer = models.ForeignKey(Customer, on_delete=models.CASCADE, db_index=True, related_name="orders")
+    drop_session = models.OneToOneField(DropSession, on_delete=models.CASCADE, db_index=True,  blank=False, null=False, related_name='order')
```

Comment:
Nit: `OneToOneField` implies `unique=True`, which in turn implies `db_index=True`.